java.lang.NoSuchFieldError: ANNOTATION_PROCESSOR_MODULE_PATH

When I use the following latest plugin config in a project using JDK8 and run `mvn rewrite:run -e`, an exception occurs.
```
            <plugin>
                <groupId>org.openrewrite.maven</groupId>
                <artifactId>rewrite-maven-plugin</artifactId>
                <version>5.0.0</version>
                <configuration>
                    <activeRecipes>
                        <recipe>org.openrewrite.java.migrate.UseJavaUtilBase64</recipe>
                    </activeRecipes>
                </configuration>
                <dependencies>
                    <dependency>
                        <groupId>org.openrewrite.recipe</groupId>
                        <artifactId>rewrite-migrate-java</artifactId>
                        <version>2.0.0</version>
                    </dependency>
                </dependencies>
            </plugin>
```

Exception:
```
Caused by: java.lang.NoSuchFieldError: ANNOTATION_PROCESSOR_MODULE_PATH
    at com.sun.tools.javac.file.Locations.initHandlers (Locations.java:1976)
    at com.sun.tools.javac.file.Locations.<init> (Locations.java:145)
    at com.sun.tools.javac.file.BaseFileManager.createLocations (BaseFileManager.java:115)
    at com.sun.tools.javac.file.BaseFileManager.<init> (BaseFileManager.java:76)
    at com.sun.tools.javac.file.JavacFileManager.<init> (JavacFileManager.java:147)
    at org.openrewrite.java.ReloadableJava8Parser$ByteArrayCapableJavacFileManager.<init> (ReloadableJava8Parser.java:319)
    at org.openrewrite.java.ReloadableJava8Parser.<init> (ReloadableJava8Parser.java:86)
    at sun.reflect.NativeConstructorAccessorImpl.newInstance0 (Native Method)
    at sun.reflect.NativeConstructorAccessorImpl.newInstance (NativeConstructorAccessorImpl.java:62)
    at sun.reflect.DelegatingConstructorAccessorImpl.newInstance (DelegatingConstructorAccessorImpl.java:45)
    at java.lang.reflect.Constructor.newInstance (Constructor.java:423)
    at org.openrewrite.java.Java8Parser$Builder.build (Java8Parser.java:138)
    at org.openrewrite.java.Java8Parser$Builder.build (Java8Parser.java:68)
    at org.openrewrite.maven.MavenMojoProjectParser.listSourceFiles (MavenMojoProjectParser.java:139)
    at org.openrewrite.maven.AbstractRewriteMojo.listResults (AbstractRewriteMojo.java:280)
    at org.openrewrite.maven.AbstractRewriteRunMojo.execute (AbstractRewriteRunMojo.java:54)
```

I think this is caused by the dependency `com.google.errorprone:javac`. And the following config can workaround this problem.
```
            <plugin>
                <groupId>org.openrewrite.maven</groupId>
                <artifactId>rewrite-maven-plugin</artifactId>
                <version>5.0.0</version>
                <configuration>
                    <activeRecipes>
                        <recipe>org.openrewrite.java.migrate.UseJavaUtilBase64</recipe>
                    </activeRecipes>
                </configuration>
                <dependencies>
                    <dependency>
                        <groupId>org.openrewrite.recipe</groupId>
                        <artifactId>rewrite-migrate-java</artifactId>
                        <version>2.0.0</version>
                        <exclusions>
                            <exclusion>
                                <groupId>com.google.errorprone</groupId>
                                <artifactId>javac</artifactId>
                            </exclusion>
                        </exclusions>

                    </dependency>
                </dependencies>
            </plugin>
```

--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -16,7 +16,7 @@ dependencies {
     testImplementation("org.projectlombok:lombok:latest.release")
 
     annotationProcessor("org.openrewrite:rewrite-templating:latest.integration")
-    implementation("com.google.errorprone:error_prone_core:2.10.0:with-dependencies") {
+    implementation("com.google.errorprone:error_prone_core:2.19.1:with-dependencies") {
         exclude("com.google.auto.service", "auto-service-annotations")
     }
 

